processor/src/main/java/org/mapstruct/ap/internal/model/source/SourceMethod.java

```diff
@@ -590,6 +592,11 @@ public boolean isBeforeMappingMethod() {
         return Executables.isBeforeMappingMethod( getExecutable() );
     }
 
+    public boolean isAbstract() {
```

Comment:
`overridesMethod` would not work for methods defined in the configuration class.  `declaringMapper.getTypeElement().getKind().isInterface()` captures default methods as well, right?